packages/apostrophe/modules/@apostrophecms/styles/lib/handlers.js

```diff
@@ -2,10 +2,22 @@ const { createId } = require('@paralleldrive/cuid2');
 
 module.exports = self => {
   return {
-    '@apostrophecms/doc:afterReplicate': {
-      async migrateLegacyGlobalPaletteFields() {
-        if (self.shouldMigrateLegacyGlobalPaletteFields) {
-          await self.migrateLegacyGlobalPaletteFields();
+    'apostrophe:modulesRegistered': {
+      warnDeprecatedPalette() {
+        const paletteModule = self.apos.modules['@apostrophecms-pro/palette'];
+        if (!paletteModule?.tasks?.['migrate-to-styles']) {
```

Comment:
would it be better to check the version of palette rather than checking if it contains the task? WDYT?